docs/design/mockups/phase-10-share/sh3-full-variant-deck.html

```
@@ -0,0 +1,430 @@
+        <li><b>#13–16</b> Square: pre-game ±foto, result ±foto (new 1:1 surface)</li>
+      </ol>
+      <h3>Punctuation = sentiment</h3>
+      <p>Warm <b>!</b> on KCVV-positive shouts (#3 #4 Goal! · #9 Rood! · #10 Gewonnen! · #15 #16). Sober <b>.</b> everywhere else (#5 #6 #7 #8 #11 #12).</p>
+      <div class="q">
+        <b>ROUND 3 — comb through the deck.</b> Feedback per card by number. Open questions I'd especially like a call on:
+        <b>(a) #8 Rood · KCVV</b> — keep it cream-sober, or make our own red card go <b>dark/dramatic</b> (register B)?
+        <b>(b) #6/#7 yellow cards</b> — is the difference (brick vs jersey accent, ± player) clear enough, or too subtle?
+        <b>(c) #14/#16 with-image squares</b> — photo placement working?
+        <b>(d)</b> anything to flip in the A/B + punctuation rule.
+        Once the deck is signed off, I'll write the per-template detail spec (Round "details") + the html-to-image capture notes, then <code>/spec</code> #2156 ready.
+      </p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)